CONTRIBUTING.CHEATSHEET.POSTGRESQL.md

```diff
@@ -0,0 +1,176 @@
+# Contributing: Cheatsheet (PostgreSQL)
+
+The [CONTRIBUTING](CONTRIBUTING.md) doc provides an extensive amount of information. Below is a summary of that info to get up-and-running quickly, with a few additional explanations.
+
+## Pre-Requisites
+
+1. Have docker installed
+1. Have `yarn` installed (see _Helpful Notes_ section below)
+1. Have the database installed and running
+
+   Test Type | Requires Running Database
+   ---         | --- |
+   Unit        | No  |
+   Integration | Yes |
+   Other       | Yes | 
+
+   Note:
+   - tests DO NOT automatically start the database they're testing against  
+   (see _Helpful Notes_ section below)
+   - if the database is not running, you may encounter an error message similar to:
+      ```bash
+      SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:23010
+      ```
+
+---
+
+## Running Tests
+This example will use _PostgreSQL_ as the database being contributed to. Replace _postgres_ with whatever dialect is specific to your changes.
+
+The basic structure is start, test, stop. Start the database, run the test script, stop the database.
+
+```bash
+yarn start-postgres            # start the database
+yarn test-integration-postgres # run the test
+yarn stop-postgres             # stop the database
+```
+
+### Different Tests
+
+Test | Description
+---- | ---
+`DIALECT=postgres yarn test` | **Type**: Full test of a database<br>**Requires**: specified dialect & running db
+`yarn test-integration` | **Type**: Partial test<br>**Requires**: running db
+`yarn test-unit` | **Type**: Partial test<br>**Requires**: no requirement can be run as-is
+`DIALECT=postgres npx mocha test/**/query-generator.test.js` | **Type**: Specific test<br>**Notice**: it calls `mocha` directly <br>**Requires**: varies based on test
+
+---
+
+## Debugging / Cheatsheet
+
+Command | Description
+--- | ---
+`console.log` | `node --inspect` is not currently utilized and would require modifications into the tests. In the meantime, use `console.log` for debugging
+`docker ps -a` | Display running containers
+`docker exec -it <container name> psql -d sequelize_test -U sequelize_test` | Connect to a running database inside the container and run psql (psql is PostgreSQL-only)
+
+### PgAdmin4
+1. Running pgadmin4 as a container (pgadmin is PostgreSQL-only)
```

Comment:
Setting up the web version of pgAdmin4 is pretty cumbersome, is there any reason to use it instead of the desktop client?

Reply:
The CONTRIBUTING.md already included the docker code to start up pgadmin4; however, I couldn't get it working.  That's why I included it here to mention how to get it working.

As for _why_ I imagine that's going to be a preference.  I know at my work, we are more limited on certain applications (or really the sockets) and think the web client would be a little less weight and bypass some of the more restrictive policies.

For my preference, I don't really use a GUI, just psql when necessary.  I do think the configuration steps here may be of use to setting up other interfaces as it demonstrates how to work between the web and docker.